packages/amplify-codegen/src/commands/models.js

```diff
@@ -87,7 +87,11 @@ async function generateModels(context) {
   appsyncLocalConfig.forEach((cfg, idx) => {
     const outPutPath = cfg.filename;
     fs.ensureFileSync(outPutPath);
-    fs.writeFileSync(outPutPath, generatedCode[idx]);
+    const contentsToWrite = [getVersionsMetadataComment(context), generatedCode[idx]].filter(function (content) {
+      return content != null;
+    });
```

Comment:
```suggestion
    const contentsToWrite = [getVersionsMetadataComment(context), generatedCode[idx]].filter(content => content);
```
You can use the lambda expression instead.